core-initrd/latest/factory/usr/lib/systemd/system/sysroot.mount

```diff
@@ -1,21 +1,16 @@
+# This file is overridden by the one created by snap-bootstrap in
+# /run/systemd/system/ since UC24, but we still need this static configuration
+# to set the unit dependencies, as even after using 'systemctl daemon-reload'
+# the dependencies of units that are queued to be activated are not changed.
+# Without this file, sysroot was mounted too late.
+#
+# See https://github.com/systemd/systemd/issues/23034 for details.
 [Unit]
 DefaultDependencies=no
 Before=initrd-root-fs.target
```

Comment:
You could drop those two lines

Reply:
The main reason to keep this file is so these dependencies are considered when building the dependency graph, so unless I am getting something wrong these should still be here unless we force the mount at the time snap-bootstrap is running.

Reply:
That dependency should go in snap-initramfs-mounts.service